src/firefly/java/edu/caltech/ipac/firefly/server/query/tables/IpacTableFromSource.java

```diff
@@ -136,6 +144,16 @@ public void prepareTableMeta(TableMeta defaults, List<DataType> columns, ServerR
         if (type == null || type.equals(TYPE_CATALOG)) {
             UserCatalogQuery.addCatalogMeta(defaults,columns,request);
         }
+        if (request.containsParam(ServerParams.USE_UPLOADED_FILENAME_AS_TABLE_TITLE)){
+            if (request.getBooleanParam(ServerParams.USE_UPLOADED_FILENAME_AS_TABLE_TITLE)){
```

Comment:
I know.. in which case, getBooleanParam() will return false and you only care when it's true.